tests/screenshot/apps/components/FanSpeedControl.js

```diff
@@ -1,7 +1,15 @@
 import FanSpeedControl from '../../../../FanSpeedControl';
 
 const FanSpeedControlTests = [
+	<FanSpeedControl />,
+	<FanSpeedControl min={1} />,
+	<FanSpeedControl max={10} />,
+	<FanSpeedControl value={5} />,
+	<FanSpeedControl icon="fan" min={1} max={10} />,
+	<FanSpeedControl icon="fan" min={1} max={10} value={1} />,
 	<FanSpeedControl min={1} max={10} value={5} />,
```

Comment:
I think non-icon cases are not necessary. (There is not in GUI)
Instead, Please add disabled cases.